src/lib/markup/stimuli.js

```diff
@@ -1,6 +1,11 @@
 const baseStimulus = (element, prompt=false) => {
-  const class_ = (prompt) ? 'main-prompt': 'main'
-  return `<div class=${class_}>${element}</div>`
+  return (
+    `<div class='beads_container'>
```

Comment:
I thought you were getting rid of references to `beads_container`? is removing the class logic based on the prompt breaking something else?